src/components/App.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ButtonPanel from './ButtonPanel';
+import Display from './Display';
+
+const App = () => (
+  <div>
+    <Display result="0" />
+    <ButtonPanel />
+  </div>
```

Comment:
- [ ] App component should have its children components wrapped in React.Fragment short syntax:
```JS
const App = () => (
  <>
    <Display result="0" />
    <ButtonPanel />
  </>
```
![Screenshot from 2021-03-10 12-11-58](https://user-images.githubusercontent.com/30977845/110651118-e95b8280-8199-11eb-9eab-3580ac2fdca0.png)

- [ ] There is no need to pass the `result` prop to the Display component yet. That's why we can declare a default value for it inside the component.